src/main/kotlin/uk/gov/justice/digital/hmpps/incentivesapi/service/PrisonerIepLevelReviewService.kt

```diff
@@ -27,10 +27,7 @@ class PrisonerIepLevelReviewService(
 ) {
   suspend fun getPrisonerIepLevelHistory(bookingId: Long, useNomisData: Boolean = true, withDetails: Boolean = true): IepSummary {
     return if (useNomisData) {
-      val result = prisonApiService.getIEPSummaryPerPrisoner(listOf(bookingId)).first()
```

Comment:
When calling `POST:/bookings/iepSummary` it passes `withDetails=true` to [bookingService.getBookingIEPSummary](https://github.com/ministryofjustice/prison-api/blob/82a0d1a851a8221633babad51bfa21c24c8a4615/src/main/java/uk/gov/justice/hmpps/prison/api/resource/BookingResource.java#L512-L522). Prison API called `verifyBookingAccess` for the booking ID which resulted in a 404 because the DPS user did not have booking access. This caused Incentives API to return [500s in production](https://mojdt.slack.com/archives/CP8FR9XL5/p1649079873426879)

That DPS user has `ROLE_GLOBAL_SEARCH` and so `!isAllowedToViewAllPrisonerData` is false. Therefore if Incentives API uses the same Prison API endpoint which DPS used to uses and passes `withDetails` then the quick view screen will again work for all users